cms/static/sass/_variables.scss

```diff
@@ -87,6 +87,9 @@ $blue-t1: rgba($blue, 0.25);
 $blue-t2: rgba($blue, 0.50);
 $blue-t3: rgba($blue, 0.75);
 
+$uxpl-blue-base: #0079BC; // UXPL, also wcag2a compliant
+$uxpl-blue-hover-active: lighten($uxpl-blue-base, 7%); // UXPL, also wcag2a compliant
+
```

Comment:
I'd prefer to update our `$blue` and `$green` with these values, but it would change all blues and greens. Thoughts?